definitions/infra-kubernetes_cronjob/definition.yml

```diff
@@ -3,3 +3,6 @@ type: KUBERNETES_CRONJOB
 configuration:
   entityExpirationTime: DAILY
   alertable: true
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
```

Comment:
you need the `filterNRQLByEntityID: true` field.
https://github.com/newrelic/entity-definitions/blob/main/docs/entity_summary.md